stacker/context.py

```diff
@@ -41,7 +41,7 @@ class Context(object):
 
     def __init__(self, environment,  # pylint: disable-msg=too-many-arguments
                  stack_names=None,
-                 parameters=None, mappings=None, config=None,
+                 variables=None, mappings=None, config=None,
```

Comment:
I know you didn't introduce this, but why don't we just split these up on multiple lines - this seems out of place.